integration-tests/src/main/kotlin/org/stellar/anchor/platform/SepClient.kt

```diff
@@ -20,6 +21,19 @@ open class SepClient {
         .callTimeout(10, TimeUnit.MINUTES)
         .build()
   }
+  fun httpGetWithJwt(url: String, jwt: String): String? {
+    val builder =
+      Request.Builder()
+        .url("$url")
+        .header("Content-Type", "application/json")
+        .header("Authorization", "Bearer $jwt")
+        .get()
+
+    val request = builder.build()
+    val response = client.newCall(request).execute()
+    return handleResponse(response)
+  }
+  fun httpPostWithJwt(request: Any, jwt: String) {}
```

Comment:
Yes, that's the plan. Sorry about this half-way done PR. I kind of rush it in to avoid merge conflicts. This function is to be implemented and used by other functions such as withdraw.